features/parse-html-unsafe.yml.dist

```diff
@@ -2,39 +2,30 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: low
-  baseline_low_date: 2024-07-09
+  baseline: false
```

Comment:
See https://github.com/mdn/browser-compat-data/pull/27392/. This regression needs some attention, so I'll come back to this tomorrow.

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/27522 upstream to improve the notes.

Reply:
And https://github.com/mdn/browser-compat-data/pull/27523 too